tap-agent/src/agent/sender_account.rs

```diff
@@ -270,6 +404,15 @@ impl Actor for SenderAccount {
             "New SenderAccount message"
         );
         match message {
+            SenderAccountMessage::UpdateRav(rav) => {
+                state
+                    .rav_tracker
+                    .update(rav.message.allocationId, rav.message.valueAggregate);
+                let should_deny = !state.denied && state.deny_condition_reached();
+                if should_deny {
+                    state.add_to_denylist().await;
+                }
```

Comment:
Shouldn't it also have a `remove_from_denylist` here?

Reply:
ravs can only go up, only unaggregated fees and the other check can go down